### ISSUE TITLE
`git pull-request` with -a option not available

### ISSUE THREAD

**Issue by pbaranay** [github/hub#1198]
`hub` should let me assign a pull request to a user, but this option isn't available in my current installation of `hub`. This is especially strange since I can see the code for the `-a` option [in the source](https://github.com/github/hub/blob/master/commands/pull_request.go). I installed `hub` from `brew`.

```
$ hub version
git version 2.5.4 (Apple Git-61)
hub version 2.2.3
```

**Comment by mislav** [comment]
Yes, it's an edge feature still unreleased, that's why you're seeing it in source but not in latest release.

To install the edge version, try `brew upgrade hub --HEAD`. It will compile a version from master branch. Or, be patient for the next release 😉  Good luck!

**Comment by pbaranay** [comment]
Got it, thank you! :) For future reference (and so I don't make this mistake again), is there an easy way to tell which features are in source but not yet released?

**Comment by mislav** [comment]
Yes, you can use the git log:

1. Go to Releases;
2. Go to latest release;
3. Click “N commits to master since this release”.

The result is [this compare view](https://github.com/github/hub/compare/v2.2.3...master).

**Comment by pbaranay** [comment]
Awesome! Thanks so much.

**Comment by danihodovic** [comment]
When will this edge feature be released?

**Comment by mislav** [comment]
@danihodovic This feature is available via 2.3.0-prereleases available from the Releases section. There are prebuilt binaries that will likely match your system. Try one out!
